TodoApp.ConsoleApp/UI/TaskOperations.cs

```diff
@@ -40,8 +43,8 @@ public static void ReadOrWrite()
 
         static void ReadTasks()
         {
-            XMLTaskReader reader = new XMLTaskReader();
-            string path = "../../tasks.xml";
+            //XMLTaskReader reader = new XMLTaskReader();
```

Comment:
Remove all commented code.
We are using git, so if you need to see old code, you can see commit history.